src/seedu/addressbook/data/person/Address.java

```diff
@@ -12,9 +12,12 @@
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
     private boolean isPrivate;
-
+    
```

Comment:
are you not enforcing the new format with different components? 
if you are, how are you validating the input address?